apps/remote_control/lib/lexical/remote_control/code_mod/ast.ex

```diff
@@ -14,17 +12,7 @@ defmodule Lexical.RemoteControl.CodeMod.Ast do
   end
 
   def from(s) when is_binary(s) do
-    parse(s)
-  end
-
-  defp parse(s) when is_binary(s) do
-    case Sourceror.string_to_quoted(s, token_metadata: true, columns: true) do
-      {:ok, code, _comments} ->
-        {:ok, code}
-
-      error ->
-        error
-    end
+    Sourceror.parse_string(s)
```

Comment:
When Sourceror calls `string_to_quoted` internally, it sets a number of additional options that add metadata to parts of the AST and are later used with `Sourceror.get_range/1`, which I was relying on.

Reply:
This was swapped back to a call to `string_to_quoted`, but includes all the options Sourceror still requires. See f58c0008ace899567092d41fbd79a53fa3e5cf30 for more information.